openrtb3/no_bid_status_codes.go

```diff
@@ -0,0 +1,55 @@
+package openrtb3
+
+// NonBidStatusCode represents OpenRTB Loss Reason Code enumeration.
+//
+// It lists the options for an exchange to inform a bidder as to the reason why they did not win an item.
+type NonBidStatusCode int64
+
+// NonBidStatusCode options.
+//
+// Values 500+ are exchange specific values; should be communicated with buyers beforehand.
+const (
+	NoBidWinningBid                         NonBidStatusCode = -1  // Winning Bid
```

Comment:
Just worrying around  `NonBidStatusCode`. Whether it should be part of the specification. It's very internal to PubMatic's Analaytics. 
Let us discuss around it with @pm-viral-vala , you and me